api/v1beta1/disruption_types.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/DataDog/chaos-controller/utils"
 	"github.com/hashicorp/go-multierror"
 	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
nit: This import is called "apierrors" in other files, e.g., disruption_controller.go